salesforce/modules/soap/tests/test.bal

```diff
@@ -76,4 +76,6 @@ function testconvertLead() {
 @test:AfterSuite {}
 function testDeleteRecord() returns error? {
     check restClient->deleteAccount(accountId);
+    check restClient->deleteContact(contactId);
+    check restClient->deleteOpportunity(opportunityId);
 }
```

Comment:
Adding a sample on `convertLead` operation would be helpful to the new users

Reply:
Added in https://github.com/ballerina-platform/module-ballerinax-sfdc/commit/b1c997b55945a002a236eed5813068a4a9e2e61b